scripts/run_seo_audit.py

```
@@ -0,0 +1,539 @@
+    W, H = A4
+    GUCCI_BLACK = colors.HexColor("#1a1a1a")
+    GOLD        = colors.HexColor("#C6963A")
+    LIGHT_GRAY  = colors.HexColor("#F5F1EC")
+    MID_GRAY    = colors.HexColor("#CCCCCC")
+    DARK_GRAY   = colors.HexColor("#555555")
+    RED         = colors.HexColor("#C0392B")
+    ORANGE      = colors.HexColor("#E67E22")
+    GREEN       = colors.HexColor("#27AE60")
+
+    def score_color(s: float) -> object:
+        if s >= 80: return GREEN
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)